internal/clauderig/commands/restore.go

```
@@ -233,20 +237,73 @@ func chooseRestoreSafety(target string) string {
+// this stays a local .bak and is never what gets synced: the repo records only
+// the three identity fields, via devices.Account.
+//
+// An absent file is not an error — a machine that has never logged in has
+// nothing to protect.
+func backupIdentityFile(out io.Writer) error {
+	src, err := account.GlobalConfigPath()
+	if err != nil {
+		return nil // no home dir: nothing addressable to back up
+	}
+	if _, err := os.Stat(src); err != nil {
+		return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 293) but it returns nil

(nilerr)